src/sdk/common/OperationUtils.ts

```
@@ -14,14 +19,14 @@ export function toJSON(op: Partial<BaseAccountUserOperationStruct>): Promise<any
         else if (typeof val !== 'string' || !val.startsWith('0x')) {
           val = toHex(val);
         }
-        return [key, val];
+        return [key, val as string];
       })
       .reduce(
         (set, [k, v]) => ({
           ...set,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)